plugins/chat/app/models/chat_upload.rb

```diff
@@ -1,5 +1,10 @@
 # frozen_string_literal: true
 
+# TODO (martin) DEPRECATED: Remove this model once UploadReference has been
+# in place for a couple of months, 2023-04-01
+#
+# NOTE: Do not use this model anymore, chat messages are linked to uploads via
+# the UploadReference table now, just like everything else.
 class ChatUpload < ActiveRecord::Base
```

Comment:
can we raise if used?

```
class ChatUpload < ActiveRecord::Base
  deprecate *public_instance_methods(false)
end
```

Reply:
I guess that will cause issues with your  specs still checking on `ChatUpload`, but with this change you could just remove it, you are sure `ChatUpload` is not changing.

Reply:
Oh wow didn't know about that deprecate trick. Will try this out :+1: